kubernetes/overlays/catalog-process/dev-refactor/configmap.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{SERVICE_NAME}}
+  namespace: {{NAMESPACE}}
+data:
+  MOCK_FILE_MANAGER: "true"
```

Comment:
the file persistence should not be mocked in the environment, let's remove it.
(I know it will make the app crash until the merge of [this PR](https://github.com/pagopa/interop-be-monorepo/pull/198/files#diff-e3474e6583bc8796f56e5d2fbea688395e7df6a85792c6ef20b0e34b2d071261L7), but IMO it is preferable compared to thinking everything is working, but it's not 🤣 )
```suggestion
```